db/seeds.rb

```diff
@@ -123,6 +126,14 @@ def build_users
     user10 = User.create(fullname: "Joe Blow", email: "joe@gmail.com", display_name: "joe", password: "joe", role: 1, image: "joe_blow.jpg")
   end
 
+	def	build_supplier_admin
+		SupplierAdmin.create(user_id: 10, supplier_id: 1)
+	end
+
+	def build_supplier
+		Supplier.create(name: "Joe's big warehouse", slug: "joe-s-big-warehouse", description: "the best", retired: false, address: "1510 blake St. , CO" )
```

Comment:
Line is too long. [149/80]<br>Use 2 (not 1) spaces for indentation.<br>Space inside parentheses detected.<br>Tab detected.